### ISSUE TITLE
Shields of Germany

### ISSUE THREAD

**Issue by claysmalley** [ZeLonewolf/openstreetmap-americana#389]
In Germany, _Bundesautobahnen_ (federal motorways) have a white-on-blue hexagonal shield and _Bundesstraßen_ (federal highways) have a black-on-yellow fixed-width rectangular shield.

Most numbered routes in Germany are covered by `route=road` relations, but `network=*` tagging is not country-prefixed. There are a huge number of `network=*` values in Germany, with limited documentation. `BAB` clearly corresponds to _Bundesautobahnen_, but _Bundesstraßen_ do not have similar relation coverage with consistently tagged `network=*` values. A decision would need to be made on whether this tagging scheme is sufficient, and whether it is appropriate mapper feedback to support it.

<img src="https://user-images.githubusercontent.com/1732117/173170254-6ac4df09-f708-4924-b5b3-f474a1148165.svg" alt="Bundesautobahn_14_number" width="200"/>
<img src="https://user-images.githubusercontent.com/1732117/173170255-906e4782-4c6e-49f7-9bdf-a872fdfe5a8b.svg" alt="Bundesstraße_1_number" width="200"/>

**Comment by ZeLonewolf** [comment]
Note the discussion in #149.  In summary, it's not clear that the community in Germany has come to any consensus on the tagging of route relations, and at least one prominent member of the community has suggested that they might even be removed.  I would recommend that we not render Germany's highway shields until the community there has come to some level of consensus, otherwise we'd be driving the tagging scheme in one direction or another.

**Comment by Blijbol** [comment]
As one of the most prominent data consumers of road route relations, I think you should not be afraid to drive the tagging scheme in a direction that is workable for you (i.e. the relations exist and have a country network prefix).

I was initially sceptical about the benefits when people proposed creating these relations in the Netherlands, but this project has clearly demonstrated that you can do cool things with them that exceed what you can (reliably) do with the ref-tags on ways alone.

**Comment by 1ec5** [comment]
In addition to _Bundesautobahnen_ and _Bundesstraßen_, _Landesstraßen_ and _Kreisstraßen_ are also signposted, but only on kilometer markers, not directional signs. In the past, I’ve heard from German drivers that L and K numbers would be unrecognizable compared to road names. If we do mark them with shields, the shields would typically be black-on-white rectangles, which is why I think it would be potentially confusing to mark any other kind of road in Germany with that color scheme: https://github.com/osm-americana/openstreetmap-americana/pull/402#issuecomment-1155797061.

[<img width="960" height="720" alt="K 4148" src="https://github.com/user-attachments/assets/964d20b9-abe9-4447-92bf-d84ba151c409" />](https://commons.wikimedia.org/wiki/File:K_4148.JPG)

The states of Bavaria and Saxony have _Staatsstraßen_, which are equivalent to _Landesstraßen_. [In Saxony](https://de.wikipedia.org/wiki/Liste_der_Staatsstraßen_in_Sachsen_bis_zur_S_199) but not Bavaria, they’re marked by reassurance markers with a green trim:

[<img width="330" height="569" alt="S 258" src="https://github.com/user-attachments/assets/5aaf676b-502c-4f85-9f04-e19a5123b9b3" />](https://commons.wikimedia.org/wiki/File:S_258_Zwoenitz.jpg)

**Comment by ZeLonewolf** [comment]
> If we do mark them with shields

This brings up the notion that we may encounter a case where `network` and `ref` are correctly and appropriately tagged, but we desire the behavior of suppressing the shield entirely for cartography reasons.

**Comment by 1ec5** [comment]
Yes, that is part of the point of #402. In many countries including the U.S., some minor route networks have bona fide refs that are only signposted on standard street name signs rather than shields, but still suitable for navigation regardless, so rendering them with regular labels would be appropriate. However, _Landesstraßen_ and _Kreisstraßen_ numbers don’t seem to be that useful for wayfinding. In similar situations in the U.S., we’ve opted to tag [`unsigned_ref`](https://wiki.openstreetmap.org/wiki/Key:unsigned_ref) even though the number is _technically_ signposted on a milepost.

**Comment by kaneap** [comment]
In addition, Germany also has a network of touristic scenic routes, which (many? some?) have these shield-like signs: 
![Alleenstrasse Ruegen](https://upload.wikimedia.org/wikipedia/commons/4/4c/AlleenstrasseRuegen1.jpg)
 
This one is from the [German Avenue Road](https://en.wikipedia.org/wiki/German_Avenue_Road) (Alleenstraße)

The OSM wiki has a [list](https://wiki.openstreetmap.org/wiki/Germany/Ferienstra%C3%9Fen) of them alongside any route relations which exist.

**Comment by 1ec5** [comment]
> In summary, it's not clear that the community in Germany has come to any consensus on the tagging of route relations, and at least one prominent member of the community has suggested that they might even be removed.

In the parallel universe of OpenHistoricalMap, the German community [loves relations](https://mapstodon.space/@ohm/115494377587379890) more than anyone else and also favors a namespaced `network=DE:BAB` tag for the _Bundesautobahnen_. https://github.com/osm-americana/openstreetmap-americana/pull/1330#discussion_r2750471850 implements the _Bundesautobahnen_ shield in OHM Americana.
